python/apps/taiga/tests/integration/taiga/users/test_api.py

```diff
@@ -285,6 +285,25 @@ async def test_update_my_user_success(client):
     assert response.status_code == status.HTTP_200_OK, response.text
 
 
+#####################################################################
+# DELETE /my/user
+#####################################################################
+
+
+async def test_delete_user_204_success(client):
+    user = await f.create_user(username="user", is_active=True)
+
+    client.login(user)
+    response = client.delete("/my/user")
+    assert response.status_code == status.HTTP_204_NO_CONTENT, response.text
+
+
+async def test_delete_user_401_no_authenticated_user(client):
```

Comment:
It should be `test_delete_user_401_unauthorized` or `test_delete_user_401_unauthorized_user` if you want to add the "user" suffix.

Reply:
Done!